spec/factories.rb

```diff
@@ -1,5 +1,7 @@
 class User
   attr_accessor :name, :phone, :email, :address
+
+  def save! ; end
```

Comment:
What happens without this line?

Reply:
https://travis-ci.org/djezzzl/factory_trace/jobs/539705513

Reply:
Instead this, I'd define factories in a bit different way: 
`association :author, factory: :user, strategy: :build` So we're sure that when parent object is built, association will also have strategy built. 
Could you adjust it please?